R/strat_ests_barplot.R

```diff
@@ -0,0 +1,81 @@
+#' Barplot method for `summary.seroincidence.by` objects
+#'
+#' @param object a `summary.seroincidence.by` object (generated by applying the `summary()` method to the output of [estimate_scr_by()]).
+#' @param yvar the name of a stratifying variable in `object`.
+#' @param color_var [character] the name of the variable used for fill color (e.g., "Country").
+#' @param alpha transparency for the bars (1 = no transparency, 0 = fully transparent).
+#' @param CIs [logical], if `TRUE`, add CI error bars.
+#' @param title a title for the final plot.
+#' @param xlab a label for the x-axis of the final plot.
+#' @param ylab a label for the y-axis of the final plot.
+#' @param fill_lab fill label.
+#' @param fill_palette optional color palette for bar fill.
+#' @param ... unused.
+#'
+#' @return a [ggplot2::ggplot()] object.
+#' @export
+#' @keywords internal
+#'
+strat_ests_barplot <- function(
+    object,
+    yvar,
+    color_var = NULL,
+    alpha = 0.7,
+    CIs = FALSE,
```

Comment:
```suggestion
    CIs = FALSE, # nolint: object_name_linter
```
see https://lintr.r-lib.org/articles/lintr.html?q=nolint#excluding-only-some-linters for details; please use `# nolint` sparingly, but in this case it seems reasonable to use